### DISCUSSION TITLE
clog-db-admin Status

### DISCUSSION THREAD

**Discussion by rabbibotton** [rabbibotton/clog#31] [Show and tell]
Work now on bring clog-db-admin. I am forced for now to use the sqlite bindings instead of cl-dbi as some sort of bug/memory issue I have not been able to correct comes up on my M1 Mac. Eventually I will either use cl-dbi or just update to use the individual bindings for MySQL and PostgreSQL.

Start the code up with (ql:quickload :clog/tools) (clog-tools:clog-db-admin)

<img width="807" alt="clog-db-admin" src="https://user-images.githubusercontent.com/11616789/109036014-31a57b80-7697-11eb-872e-3f9e3bc680f9.png">

**Comment by aykaramba** [comment]
All of this just full of pure COOL. Thank you kindly for the inspiration to learn Lisp.

**Reply by rabbibotton** [reply]
Well the latest version now lets you edit records etc. :) It is cool stuff Lisp and CLOG :)
